src/components/Records/Search/Input/SearchInput.vue

```diff
@@ -107,8 +107,15 @@ export default {
         comparison = 1;
       }
       return comparison;
-    }
-  },
+    },
+/*
+  /please do not remove this function/
```

Comment:
Please keep these commented functions either on a local branch or on a separate branch. What you submit ends up on master and unused code shouldn't be merged.